3rdParty/CMakeLists.txt

```diff
@@ -36,13 +36,17 @@ function (add_snappy)
     # snappy
     add_c_flags_if_supported(CMAKE_C_FLAGS -Wno-suggest-override -Wno-sign-compare)
     add_cxx_flags_if_supported(CMAKE_CXX_FLAGS -Wno-suggest-override -Wno-sign-compare)
-    set(SNAPPY_VERSION "1.1.8")
+    set(SNAPPY_VERSION "1.1.9")
     set(SNAPPY_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/snappy/snappy-${SNAPPY_VERSION}")
     set(SNAPPY_SOURCE_DIR "${SNAPPY_SOURCE_DIR}" PARENT_SCOPE)
     set(SNAPPY_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/snappy/snappy-${SNAPPY_VERSION}" PARENT_SCOPE)
     set(SNAPPY_LIB "snappy" PARENT_SCOPE)
-    set(SNAPPY_BUILD_TESTS OFF CACHE BOOL "disable Snappy tests")
+    set(SNAPPY_FUZZING_BUILD OFF CACHE BOOL "build Snappy for fuzzing")
+    set(SNAPPY_BUILD_TESTS OFF CACHE BOOL "disable Snappy tests" FORCE)
+    set(SNAPPY_BUILD_BENCHMARKS OFF CACHE BOOL "disable Snappy benchmarks" FORCE)
     set(SNAPPY_INSTALL OFF CACHE BOOL "disable Snappy installation")
+    set(SNAPPY_REQUIRE_AVX ON CACHE BOOL "target processors with AVX support" FORCE)
```

Comment:
hm, though *our* packages will have AVX etc from now on, this will permanently trash the ability to build without?

Reply:
@dothebart: yes, unless the CMakeLists.txt file(s) are modified.
See https://github.com/arangodb/docs/blob/main/3.9/release-notes-upgrading-changes39.md#architecture-requirements